lib/node_modules/@stdlib/stats/base/snanmeanpn/benchmark/c/benchmark.length.c

```diff
@@ -94,19 +94,24 @@ static float rand_float( void ) {
 * @param len          array length
 * @return             elapsed time in seconds
 */
-static double benchmark( int iterations, int len ) {
+static double benchmark1( int iterations, int len ) {
 	double elapsed;
 	float x[ len ];
 	float v;
 	double t;
 	int i;
 
 	for ( i = 0; i < len; i++ ) {
-		x[ i ] = ( rand_float()*20000.0f ) - 10000.0f;
+		if ( rand_float() < 0.2 ) {
+			x[ i ] = 0.0 / 0.0; // NaN
+		} else {
+			x[ i ] = ( rand_float() * 20000.0 ) - 10000.0;
```

Comment:
```suggestion
		if ( rand_float() < 0.2f ) {
			x[ i ] = 0.0f / 0.0f; // NaN
		} else {
			x[ i ] = ( rand_float() * 20000.0f ) - 10000.0f;
```

This applies here and below.